msal/tests/Test.MSAL.NET.Unit.net45/CoreTests/PlatformProxyFactoryTests.cs

```diff
@@ -43,5 +43,26 @@ public void PlatformProxyFactoryCachesTheProxy()
             // Assert
             Assert.IsTrue(proxy1 == proxy2);
         }
+
+        [TestMethod]
+        public void PlatformProxyFactoryReturnsInstances()
+        {
+            // Arrange
+            var factory = PlatformProxyFactory.GetPlatformProxy();
+
+            // Act and Assert
+            Assert.AreNotEqual(
+                factory.CreateLegacyCachePersistence(),
+                factory.CreateLegacyCachePersistence());
+
+            Assert.AreNotEqual(
+                factory.CreateTokenCacheAccessor(),
+                factory.CreateTokenCacheAccessor());
+
+            Assert.AreEqual(
```

Comment:
Do you want "AreEqual" here or "AreSame" (which checks for the same reference)

Reply:
fixed.